Huion_Inspiroy_Q11K/descriptors.txt

```diff
@@ -11,8 +11,8 @@ Device Descriptor:
   idVendor           0x256c 
   idProduct          0x006e 
   bcdDevice            0.00
-  iManufacturer           5 (error)
-  iProduct                6 (error)
+  iManufacturer           5 Huion
+  iProduct                6 Inspiroy Q11K
```

Comment:
Same here, I don't think we should touch these. We need to know what the tablet reports exactly.